.circleci/config.yml

```diff
@@ -517,6 +563,14 @@ jobs:
 
   spm-revenuecat-ui-ios-17:
     <<: *base-job
+    parameters:
+      xcode_version:
+        type: string
+        default: '15.2'
```

Comment:
nit but we have this as `15.2.0` in line 652

Reply:
yeah I noticed it but didn't want to modify the version numbers since they weren't from this PR, but maybe I just got lazy. updating